components/Image/index.jsx

```diff
@@ -74,14 +57,15 @@ class Image extends React.Component {
     return (
       <span
         aria-label={alt}
-        className="img"
+        className={`img lightbox ${this.state.lightbox ? 'open' : 'closed'}`}
         onClick={() => this.toggle()}
         onKeyDown={this.handleKey}
         role={'button'}
         tabIndex={0}
       >
-        <img {...props} alt={alt} loading={lazy ? 'lazy' : ''} />
```

Comment:
This renders the image as `opacity: 0`, etc. Does it need a different closed state?

![image](https://user-images.githubusercontent.com/451488/214912923-2ad842cb-346b-4ea2-9b2b-e93e9a9d1dd5.png)

Reply:
Only happens in the preview app because we include the `bundle-hub2.css` stylesheet, which in turn has the old `.lightbox` styles baked in. If you comment that out you can see how this will actually look/work once we've merged this PR and updated the RDMD dep in ReadMe!

Reply:
Oh, right.